inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/batch_to_space_ref.cl

```diff
@@ -84,5 +89,11 @@ KERNEL(batch_to_space_ref)(const __global INPUT0_TYPE* input, __global OUTPUT_TY
                               y * OUTPUT_Y_PITCH +
                               x;
 
-    output[output_index] = ACTIVATION(input[input_index], ACTIVATION_PARAMS);
+    INPUT0_TYPE result = input[input_index];
```

Comment:
Since b_fs_yx_fsv16 is blocked layout it has more difficult index calculation function, so current input and output indices maybe incorrect. I propose to use INPUT0_GET_INDEX and OUTPUT_GET_INDEX macros instead.